src/backend/distributed/metadata/dependency.c

```diff
@@ -1056,6 +1048,27 @@ CreateObjectAddressDependencyDef(Oid classId, Oid objectId)
 }
 
 
+/*
+ * CreateObjectAddressDependencyDefList is a wrapper function for
+ * CreateObjectAddressDependencyDef to operate on a list of relation oids,
+ * instead of a single oid.
+ */
+static List *
+CreateObjectAddressDependencyDefList(Oid classId, List *oidList)
```

Comment:
minor: `oidList` -> `objectIdList` and `oid` -> `objectId`

so that we follow the more generic naming convention